mesmerize/viewer/modules/roi_manager.py

```diff
@@ -85,6 +88,16 @@ def __init__(self, parent, viewer_reference):
             ]:
             w.clicked.connect(self.set_curveplot_datatype)
 
+        self.ui.btnExport.clicked.connect(self.export_curves)
+
+    def export_curves(self):
+        """Export raw curves when Export Curves button clicked"""
+        save_path = QtGui.QFileDialog.getSaveFileName(self, "Save curves", "", "csv files (*.csv)")[0]
```

Comment:
use the `@use_save_file_dialog` instead:
http://docs.mesmerizelab.org/en/master/api_reference/common.html#mesmerize.common.qdialogs.use_save_file_dialog

Example:

```python
from ...common.qdialogs import use_save_file_dialog
# I think 3 dots will be needed to import from `common.qdialogs`, pycharm would tell you
# if this doesn't import, you might need 4:
# from ....common.qdialogs import use_save_file_dialog

@use_save_file_dialog('Export data to csv file', ext=['csv'])
def export_curves(self, path, *args):
    # stuff here
    df.to_csv(path, index=None)
    # more stuff if necessary
```

Reply:
Thanks this looks great will try it out!